### ISSUE TITLE
0.16.0's analyze-profile --html output is exceedingly large

### ISSUE THREAD

**Issue by jin** [bazelbuild/bazel#5648]
### Description of the problem / feature request:

In 0.16.0rc1, the HTML profile generated from `bazel analyze-profile --html` is exceedingly large, containing timestamp `<div>` elements like the following:

```
<div style='position: absolute; left: 10505400px; top: -20px; width: 1px; height: 40px; border-left: 1px solid rgba(128,128,128,1.000000)'></div>
<div style="position:absolute; left:10505454px; top:-17px; width:50px; height:10px">210109s</div>
<div style='position: absolute; left: 10505450px; top: -20px; width: 1px; height: 40px; border-left: 1px solid rgba(128,128,128,1.000000)'></div>
<div style="position:absolute; left:10505504px; top:-17px; width:50px; height:10px">210110s</div>
<div style='position: absolute; left: 10505500px; top: -20px; width: 1px; height: 40px; border-left: 1px solid rgba(128,128,128,1.000000)'></div>
```

A HTML profile on `//examples/cpp:hello-world` is 48MB and has 420511 lines:

```
$ wc -l 0.16.0rc1.prof.html
  420511 0.16.0rc1.prof.html
$ ls -lh 0.16.0rc1.prof.html
-rw-r--r--  1 jingwen  5001    48M Jul 20 23:22 0.16.0rc1.prof.html
```

For comparison, the same profile generated from 0.15.2 is 279 lines:

```
$ wc -l 0.15.2.prof.html
     279 0.15.2.prof.html
```

### Bugs: what's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

Build any target using Bazel 0.16.0 RC with `--profile=prof.dat`, and `bazel analyze-profile --html prof.dat`

### What operating system are you running Bazel on?

macOS

### What's the output of `bazel info release`?

0.16.0rc1

**Comment by jin** [comment]
Bisected to 15b8c259db111012b4642287172cb4d1d82151f3

**Comment by ulfjack** [comment]
Fixed in f45c22407e6b00fcba706eb62141cb9036bd38d7. @lberki
